### PR TITLE
Parity transactions methods

### PR DESCRIPTION
**Pull request by ArtemiusUA**
### What was wrong?

Parity methods were not implemented: parity_allTransactions, parity_futureTransactions, parity_pendingTransactions, parity_pendingTransactionsStats, parity_localTransactions

### How was it fixed?

Added methods, added formatters.

#### Cute Animal Picture

Put a link to a cute animal picture inside the parenthesis-->

### CODE REVIEW

File: web3/middleware/pythonic.py [hunk 1]
```diff
@@ -360,5 +364,10 @@ def to_hexbytes(num_bytes, val, variable_length=False):
         'evm_snapshot': hex_to_integer,
         # Net
         'net_peerCount': to_integer_if_hex,
+        # Parity
```
Comment (pipermerriam) [inline web3/middleware/pythonic.py]: @carver and @dylanjw what do you think about isolating these into their own parity specific middleware.

Comment (carver) [reply web3/middleware/pythonic.py]: Yeah, that seems like the right way to go. We could extend `Parity.attach(w3)` to insert the parity middleware for you.

**Comment by ankitchiplunkar** [comment]
@ArtemiusUA is this PR still being worked on?
If not I can take a swing at it.

**Comment by carver** [comment]
> @ArtemiusUA is this PR still being worked on?
> If not I can take a swing at it.

Given the time lag, it seems fair to assume it's okay to pick it up.

**Comment by kclowes** [comment]
Closing since Parity is no longer maintained.

**Pull request closed without merging**